kuma/static/js/libs/ckeditor/source/build.sh

```diff
@@ -60,6 +60,13 @@ function download_plugin
 	git checkout $2
 	cd -
 	rm -rf plugins/$1/.git
+
+	# If the plugin is in a subdirectory, move it where it belongs
+
+	if [ ! -f plugins/$1/plugin.js ]; then
```

Comment:
OK. I was trying to be more convenient for future use, but that's fine.

As for the language files -- I was under the impression that we didn't include the plugin code that gets downloaded by the build tool during the build process, which these files are.